upi/vsphere/bootstrap/main.tf

```diff
@@ -0,0 +1,66 @@
+data "vsphere_datacenter" "dc" {
+  name = "${var.datacenter}"
+}
+
+data "vsphere_compute_cluster" "compute_cluster" {
+  name          = "${var.cluster}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_datastore" "datastore" {
+  name          = "${var.datastore}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_network" "network" {
+  name          = "${var.vm_private_network}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+data "vsphere_virtual_machine" "template" {
+  name          = "${var.vm_template}"
+  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+}
+
+resource "vsphere_resource_pool" "resource_pool" {
+  name                    = "${var.resource_pool}"
+  parent_resource_pool_id = "${data.vsphere_compute_cluster.compute_cluster.resource_pool_id}"
+}
+
+resource "vsphere_virtual_machine" "vm" {
+  name             = "bootstrap-0"
+  resource_pool_id = "${vsphere_resource_pool.resource_pool.id}"
+  datastore_id     = "${data.vsphere_datastore.datastore.id}"
+  num_cpus         = 4
+  memory           = 8192
```

Comment:
bootstrap machine can mostly likely be 2048 memory and 2 cpus (same as libvirt).

Reply:
Agreed! Parameterized CPU and memory. I haven't started writing that section yet. But, I'll pass the appropriate sizing.